apps/protocol-frontend/src/components/AttestationsTable.tsx

```diff
@@ -56,10 +57,17 @@ const AttestationsTable = ({
 }) => {
   const { userData } = useUser();
 
-  const nonUserContributions = _.filter(contributionsData, function (a) {
-    return a.user.id !== userData?.id;
+  const filteredMintedContributions = _.filter(contributionsData, function (a) {
```

Comment:
A better name may be mintedContributions. If there isn't another variable named minted contributions